mshadow/packet/sse-inl.h

```diff
@@ -17,7 +17,7 @@ template<>
 struct Packet<float, kSSE2> {
  public:
   /*! \brief number of float in vector */
-  static const index_t kSize = 4;
```

Comment:
Causes link error in newer versions of gcc

See:  http://stackoverflow.com/questions/5391973/undefined-reference-to-static-const-int